### ISSUE TITLE
Google Places autocomplete field not functional when more than one form on a page

### ISSUE THREAD

**Issue by russback** [verbb/formie#342]
**Description**
I've set up a form that works fine with Google Places enabled. However when I add a second form on the same page, the lookup no longer appears to happen. 

I noticed this after I in effect added a second form to a page by putting a newsletter signup form in the site footer. When I remove that form in the footer, the lookup works again on the original form.

### Form settings

- Multi-page form: Yes
- Submission Method: Ajax
- Client-side Validation: Yes
- Custom Form Templates: No

**Additional info**
- Plugin version: 1.3.16.1
- Craft version: 3.6.4.1
- Multi-site: Yes

**Comment by engram-design** [comment]
Pretty sure this should be fixed in https://github.com/verbb/formie/commit/554791f8a47c7d57fb8aca43ca829ffe1b78ed08 To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.3.16.1",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Should be fixed in [1.3.17](https://github.com/verbb/formie/releases/tag/1.3.17)
